src/Validator/ConstraintValidatorInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Validator;
+
+use ApiPlatform\Core\Validator\Exception\ValidationException;
+use Symfony\Component\Validator\Constraint;
```

Comment:
This creates a constraint to the Symfony validator outside the Symfony bridge. This defeats the purpose of this abstraction and will cause issue to support other validator libraries (and manual validation).

Wouldn't it be better to just use the Symfony Validator directly for this kind of use cases?